functions/TextEnrichment/__init__.py

```diff
@@ -38,6 +38,8 @@
 queueName = os.environ["EMBEDDINGS_QUEUE"]
 azure_ai_translation_domain = os.environ["AZURE_AI_TRANSLATION_DOMAIN"]
 azure_ai_text_analytics_domain = os.environ["AZURE_AI_TEXT_ANALYTICS_DOMAIN"]
+endpoint_region = os.environ["ENRICHMENT_LOCATION"]

```

Comment:
What sets these new parameters? I don't see any TF or script updates to populate these new variables.

Reply:
line 151 in functions.tf. They are preexisting. Probably implemented after the original code. Reduce/Reuse/Recycle